build/cmake/init.cmake

```
@@ -36,6 +36,26 @@ if(MSVC)
               "Flags used by the CXX compiler during ${cfg_upper} builds." FORCE)
         endif()
     endforeach()
+    
+    if(wxUSE_OPTIMISE)
+        set(MVC_PREFERRED_RELEASE_FLAGS " /LTCG /OPT:REF,ICF /DEBUG")
+        wx_string_append(CMAKE_EXE_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
+        wx_string_append(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
+        wx_string_append(CMAKE_CXX_FLAGS_RELEASE " -DNDEBUG /Ox /Ob2 /Oi /Ot /Oy /GS- /Gy /GR- /GL /Gw")
+    endif()
+    
+    set(MSC_PDB_FLAG " /DEBUG") 
+    wx_string_append(CMAKE_EXE_LINKER_FLAGS_RELEASE "${MSC_PDB_FLAG}")
+    wx_string_append(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${MSC_PDB_FLAG}")
+elseif(CMAKE_COMPILER_IS_GNUCC)
+    if(wxUSE_OPTIMISE)
+        set(GCC_STRIPPING_FLAG " -g") 
+        set(GCC_PREFERRED_RELEASE_FLAGS "-O2 -DNDEBUG -fomit-frame-pointer ${GCC_STRIPPING_FLAG}")
```

```suggestion
        set(GCC_PREFERRED_RELEASE_FLAGS " -O2 -fomit-frame-pointer ${GCC_STRIPPING_FLAG}")
```
Space to make sure no values are concatenated.
` -DNDEBUG` is enabled by default.

---

Done